src/test/java/org/openelisglobal/patient/PatientServiceTest.java

```diff
@@ -25,7 +25,7 @@
 @ContextConfiguration(classes = { BaseTestConfig.class, PatientTestConfig.class })
 @TestPropertySource("classpath:common.properties")
 @ActiveProfiles("test")
-public class PatientServiceTest {
```

Comment:
Why this change ?

Reply:
Hi, I am honestly sorry for the late reply. Here, I want to check if I was able to integrate any changes to this application or not. 
Integrating bearer token authentication into the patient's service test enhances security, ensures compliance, and protects patient data. It's a significant contribution that strengthens access control and aligns with industry standards, benefiting the team's expertise and future projects. It's the initial stage of my contribution. It will take a couple of days to figure out how to make any further changes. My first milestone will be to modify the OpenELIS application code to interact with the search engine and implement functionality to index data into the search engine when records are created, updated, or deleted in the OpenELIS database.